tools/c7n_gcp/c7n_gcp/resources/logging.py

```diff
@@ -11,32 +11,16 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
+from c7n.utils import local_session, type_schema
+
+from c7n_gcp.actions import MethodAction
 from c7n_gcp.provider import resources
 from c7n_gcp.query import QueryResourceManager, TypeInfo
 
 # TODO .. folder, billing account, org sink
 # how to map them given a project level root entity sans use of c7n-org
 
 
-@resources.register('logsink')
-class LogSink(QueryResourceManager):
-
-    class resource_type(TypeInfo):
-        service = 'logging'
-        version = 'v2'
-        component = 'projects.sinks'
-        enum_spec = ('list', 'sinks[]', None)
-        scope_key = 'parent'
-        scope_template = "projects/{}"
-        id = "name"
-
-        @staticmethod
-        def get(client, resource_info):
-            return client.execute_query('get', {
-                'sinkName': 'projects/{project_id}/sinks/{name}'.format(
-                    **resource_info)})
-
-
 @resources.register('log-project-sink')
```

Comment:
i've been wondering with the gcp resources if we need to move to a hierarchy to reflect the deep nesting thats exposed .. ie .  logsink.project   and logsink.org .. also just trying to clarify the removal of the `logsink` resource ?

Reply:
@kapilt No such hierarchy is needed..

And I removed "logsink" resource cause it was exact copy of "log-project-sink" resource..
I left second one cause it was closer to common naming convention here..